src/context.rs

```diff
@@ -0,0 +1,103 @@
+use std::collections::HashMap;
+
+use serde::Serialize;
+use windows_core::{BSTR, VARIANT};
+
+use crate::{WMIConnection, WMIResult};
+
+#[derive(Debug, PartialEq, Serialize, Clone)]
+#[serde(untagged)]
```

Comment:
Why the `untagged`?

Reply:
🤦‍♂️ Copy + Pasted the `Variant` enum and forgot to update. Thanks!